<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rediget</class>
 <widget class="QDialog" name="rediget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rediģēt personu un tās kontaktinformāciju</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Parādīt datus</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>680</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Izvēlies personu, kuru vēlies rediģēt:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vards">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>650</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Uzvards">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>690</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>650</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Vārds</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>690</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Uzvārds</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ievadi jauno vārdu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ievadi jauno uzvārdu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="red_vardu">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rediģēt vārdu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vards_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="uzvards">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QComboBox" name="paraditpersonas">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tel_nr">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ievadi jauno telefona nummuru</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ievadi jauno dzimšanas datumu</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dz_dat">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>5</month>
     <day>25</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Izvēlies jauno līgumu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Izvēlies jauno amatu</string>
   </property>
  </widget>
  <widget class="QComboBox" name="amats">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="lig">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="adrese">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Izvēlies jauno adresi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rediget_uzvardu">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rediģēt uzvārdu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rediget_telefona_nummuru">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rediģēt telefona nummuru</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rediget_dzimsanas_datumu">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rediģēt dzimšanas datumu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rediget_amatu">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rediģēt amatu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rediget_ligumu">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rediģēt līgumu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rediget_adresi">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Rediģēt adresi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
